pkgs/development/tools/parsing/tree-sitter/grammars/grammar-sources.nix

```diff
@@ -0,0 +1,771 @@
+{ lib }:
+{
```

Comment:
Is there any reason this needs to be a nix file instead of staying json? I believe we would get better eval performance just reading json.

Reply:
The current set of definitions are all quite vanilla. Declaring as nix rather than json allows [other sources](https://github.com/NixOS/nixpkgs/blob/eae9cfbad2793d00b2f0c6039734cbb455b64d91/pkgs/development/tools/parsing/tree-sitter/grammars/README.md#supported-sources) or [build behaviour](https://github.com/NixOS/nixpkgs/blob/eae9cfbad2793d00b2f0c6039734cbb455b64d91/pkgs/development/tools/parsing/tree-sitter/grammars/README.md#modifying-build-behaviour) to be declared when needed without fragmenting these definitions.